Backend/NoCode.FlowerShop.Api/Controllers/AdministratorsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MapsterMapper;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using NoCode.FlowerShop.Application.Administrators.Authentication.Login;
+using NoCode.FlowerShop.Contracts.Employees.Authentication;
+
+namespace NoCode.FlowerShop.Api.Controllers;
+
+[Route("administrators")]
+public class AdministratorsController : ApiController
+{
+    private readonly IMapper _mapper;
+    private readonly ISender _mediator;
+
+    public AdministratorsController(ISender mediator, IMapper mapper)
+    {
+        _mediator = mediator;
+        _mapper = mapper;
+    }
+
+    [HttpPost("login")]
```

Comment:
Why POST and not GET here?

Reply:
Initially I wanted to have GET as well, but after some research there are reasons why it might be a bad idea:
https://security.stackexchange.com/a/147191

Reply:
These are valid reasons. I agree that it should be a POST request in this case.